Integrations/URLHaus/URLHaus.py

```diff
@@ -0,0 +1,493 @@
+        return_error('Only accepting MD5 or SHA256 hash types')
+
+    try:
+        file_information = query_payload_information(hash_type, hash).json()
+
+        if file_information['query_status'] == 'ok' and file_information['md5_hash']:
+            # URLhaus output
+            first_seen = reformat_date(file_information.get('firstseen'))
+            last_seen = reformat_date(file_information.get('lastseen'))
+
+            urlhaus_data = {
+                'MD5': file_information.get('md5_hash', ''),
+                'SHA256': file_information.get('sha256_hash', ''),
+                'Type': file_information.get('file_type', ''),
+                'Size': int(file_information.get('file_size', '')),
+                'Signature': file_information.get('signature', ''),
+                'FirstSeen': first_seen,
+                'LastSeen': last_seen,
+                'DownloadLink': file_information.get('urlhaus_download', ''),
+                'URL': file_information.get('urls', [])
+            }
+
+            virus_total_data = file_information.get('virustotal')
+            if virus_total_data:
+                urlhaus_data['VirusTotal'] = {
+                    'Percent': float(file_information.get('virustotal', {'percent': 0})['percent']),
+                    'Link': file_information.get('virustotal', {'link': ''})['link']
+                }
+
+            ec = {
+                'File': {
+                    'Size': urlhaus_data['Size'],
+                    'MD5': urlhaus_data['MD5'],
+                    'SHA256': urlhaus_data['SHA256']
+                },
+                'URLhaus.File(val.MD5 && val.MD5 === obj.MD5)': urlhaus_data
+            }
+
+            human_readable = tableToMarkdown(f'URLhaus reputation for {hash_type.upper()} : {hash}',
+                                             {
+                                                 'URLhaus link': urlhaus_data['DownloadLink'],
+                                                 'Signature': urlhaus_data['Signature'],
+                                                 'MD5': urlhaus_data['MD5'],
+                                                 'SHA256': urlhaus_data['SHA256'],
+                                                 'First seen': first_seen,
+                                                 'Last seen': last_seen
+                                             })
+            demisto.results({
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['json'],
+                'Contents': file_information,
+                'HumanReadable': human_readable,
+                'HumanReadableFormat': formats['markdown'],
+                'EntryContext': ec
+            })
+        elif (file_information['query_status'] == 'ok' and not file_information['md5_hash']) or \
+                file_information['query_status'] == 'no_results':
+            human_readable = f'## URLhaus reputation for {hash_type.upper()} : {hash}\n' \
```

Comment:
what happens here with sha256? can we have it?

Reply:
What do you mean?